PubSub/src/Message.php

```diff
@@ -193,6 +204,23 @@ public function ackId()
         return $this->ackId;
     }
 
+    /**
+     * Get the delivery attempt count.
+     *
+     * If a DeadLetterPolicy is not set on the subscription, this will be 0.
+     *
+     * Example:
+     * ```
+     * echo $message->deliveryAttempt();
+     * ```
+     *
+     * @return int|null
```

Comment:
Shouldn't this always be "int"? The method needs to return 0 if no deliveryAttempt has been set. So "null" should not be returned.